<launch>

  <remap from="joy" to="hydra_joy" />

  <include file="$(find robo_surrogate)/launch/include/head_pointer.launch.xml"/>

  <include file="$(find robo_surrogate)/launch/include/arm_mover.launch.xml">
    <arg name="command_topic" value="arm_controller/cartesian_pose/command"/>
    <arg name="tracked_frame" value="hydra_right_pivot"/>
    <arg name="deadman_button" default="10"/> <!-- use 3 for hyrdra? or 11?-->
  </include>

</launch>
